src/components/ItemList.svelte

```diff
@@ -34,6 +44,7 @@
   }
 
   $: filteredItems = paginatedItems[$currentPage - 1];
+  $: filterItems(filterTerm, showExpired);
```

Comment:
```suggestion
  $: filteredItems = paginatedItems[$currentPage - 1];
  $: filterItems(filterTerm, showExpired);
```

Do we need both of these? Could we just move `filteredItems = paginatedItems[$currentPage - 1];` into the filterItems function above?